pythran/pythonic/include/numpy/random/generator.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef PYTHONIC_INCLUDE_NUMPY_RANDOM_GENERATOR_HPP
+#define PYTHONIC_INCLUDE_NUMPY_RANDOM_GENERATOR_HPP
+
+#include <random>
+
+namespace pythonic
+{
+  namespace numpy
+  {
+    namespace random
+    {
+      namespace details
+      {
+        // Found by reading Numpy source code RNG values
+        // in (random/mtrand/randomkit.c),
+        // but the random generator type is not explicitly mentioned.
+        using default_numpy_generator_t = std::mt19937;
+        default_numpy_generator_t generator;
```

Comment:
Looking at numpy.random, all of them share the same generator with a sequence of value (I guess they are seed).
Also, it will avoid extra object creation cost no?